src/test/java/core/basesyntax/service/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.nio.file.InvalidPathException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ReaderServiceImplTest {
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String LINE_OK = "b,banana,100";
+    private static final String PATH_TO_EMPTY = "src/test/java/core/basesyntax/empty.txt";
+    private static final String PATH_TO_FRUITS = "src/test/java/core/basesyntax/fruits.txt";
+    private static final String WRONG_PATH = " ";
+    private static ReaderService readerService;
+
+    @BeforeAll
+    static void beforeAll() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    void read_Empty_File_isOk() {
+        List<String> listOfDataFromFile = readerService.getListOfDataFromFile(PATH_TO_EMPTY);
+        assertEquals(new ArrayList<>(), listOfDataFromFile);
+    }
+
+    @Test
+    void read_Data_isOk() {
```

Comment:
```suggestion
    void read_data_ok() {
```
Please review all your methods and rename them by name test method conventions